### ISSUE TITLE
Missing AzureKV config

### ISSUE THREAD

**Issue by cl0udguru** [banzaicloud/bank-vaults#161]
**Describe the bug**
When using azure bank-vaults is unable to connect to azure

**To Reproduce**

```
  unsealConfig:
    azure:
      keyVaultName: "azuretestkv"
```

```
time="2018-09-19T07:51:39Z" level=fatal msg="error creating kv store: error creating Azure Key Vault kv store: parameter 'clientID' cannot be empty"
```
**Expected behavior**
Ability to set clientID and clientSecret in unsealConfig

**Comment by bonifaido** [comment]
Hi @cl0udguru, can't you just assign the proper IAM role to the node running the Vault instance? https://github.com/banzaicloud/bank-vaults#azure 
I think writing down the clientID or clientSecret in the configuration leaves those values in plain-text.

**Comment by cl0udguru** [comment]
Hi @bonifaido looked into this its only possible with the use of Managed Service identify however the app needs to be aware. I think the method you use for the Alibaba Cloud would also apply for Azure

**Comment by bonifaido** [comment]
Hm, okay I will try to put those values at least into secrets.

**Comment by bonifaido** [comment]
Pushed a fix for this, it is already under review: #162

**Comment by bonifaido** [comment]
Just released https://github.com/banzaicloud/bank-vaults/releases/tag/0.2.24

The Docker images will be available soon @cl0udguru

**Comment by bonifaido** [comment]
This is hopefully fixed now with #166, the docker image is 0.2.26.
